cmd/proxy-server/main.go

```diff
@@ -581,6 +594,40 @@ func refreshTokenHandler(client pb.TenantServiceClient, log *logrus.Entry) http.
 	})
 }
 
+// refreshAdminTokenHandler refreshes an admin token
+func refreshAdminTokenHandler(client pb.AuthServiceClient, log *logrus.Entry) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		log.Info("Refreshing admin token!")
+		var input token.AdminToken
+		err := json.NewDecoder(r.Body).Decode(&input)
+		if err != nil {
+			log.WithError(err).Error("decoding admin token pair")
+			http.Error(w, "decoding admin token pair", http.StatusInternalServerError)
+			return
```

Comment:
`karavictl` is expecting the web.JSONErrorResponse body on an error. `cmd/karavictl/cmd/api/api.go` guarantees this but we should still use it in this handler on an error.

Reply:
fixed